### PR TITLE
Switch to eslint

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "wikimedia",
+	"env": {
+		"browser": true
```
Comment (Krinkle) [inline .eslintrc.json]: "node", not "browser".

**Comment by Krinkle** [comment]
The build is failing since grunt-eslint requires Node 4 or higher as of v19.0.0. I'll make sure Node 4 is supported, and will drop support for Node v0.10. However I'd like to keep support for v0.12 for a little while longer until we go v2.0.

**Comment by edg2s** [comment]
should I change to 18.x?

**Comment by Krinkle** [comment]
@edg2s Might work, yeah.

**Comment by edg2s** [comment]
Seems to work

**Pull request merged**